libbeat/template/config.go

```diff
@@ -48,5 +53,8 @@ func DefaultConfig() TemplateConfig {
 		Enabled: true,
 		Fields:  "",
 		Order:   1,
+
+		DefaultPipelineEnabled: true,
+		DefaultPipelineName:    "beats_default_pipeline",
```

Comment:
I think the pipeline name should have version number in it.

Reply:
And I like the idea of using the beat's name rather than "beats".

Reply:
> I think the pipeline name should have version number in it.

Good catch, definitely. We'll need it when running multiple versions of Beats.

> And I like the idea of using the beat's name rather than "beats".

👍 - now the default pipeline for all Beats would be the same, but in the future this might well diverge.